mlflow/transformers/__init__.py

```diff
@@ -1077,6 +1080,13 @@ def _load_model(path: str, flavor_config, return_type: str, device=None, **kwarg
             flavor_conf=flavor_config, accelerate_conf=accelerate_model_conf, device=device
         )
 
+    # Load and apply PEFT adaptor if saved
+    if peft_adapter_dir := flavor_config.get(FlavorKey.PEFT, None):
```

Comment:
Shall we use `FLAVOR_KEY` or `flavor_key` instead of the camel case?

Reply:
It's constant but still a class so the convention is to use the camel case I guess?